pkg/storaged/stratis-details.jsx

```diff
@@ -431,6 +435,10 @@ export const StratisPoolDetails = ({ client, pool }) => {
     }
 
     const use = pool.TotalPhysicalUsed[0] && [Number(pool.TotalPhysicalUsed[1]), Number(pool.TotalPhysicalSize)];
+    const no_ipc_requests = _("Pool is in a degraded state. Some actions are unavailable.");
+    const no_pool_changes = _("Pool is in a degraded state. Maintainance operations are not possible. Please migrate the data to a new pool");
+    const alert_msg = pool.AvailableActions === "no_ipc_requests" ? no_ipc_requests : (pool.AvailableActions === "no_pool_changes" ? no_pool_changes : null);
```

Comment:
Should we have a generic alert when AvailableActions is not "fully_functional" but also not one of the values we recognize explicitly?  Maybe just "Pool is in a degraded state."

Reply:
There is currently in the API no such state. We better need a way to query the available error codes from the API in the tests, and if some new arrives it should notify us.

Reply:
Sorry, I mean "fully_operational".  That is the value of AvailableActions when the pool is not degraded in any way.